cmd/buildah/addcopy.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	"github.com/containers/buildah"
+	"github.com/containers/buildah/internal/util"
```

Comment:
Could we use "iutil" here and in the other files just to make it easier to search through the whole project?  I realize you did it elsewhere to avoid the collision, but it would be good to have it the same everywhere.

Reply:
I will let @nalind  decide, since we have not done this with prior conflicts.

Reply:
Like alphabetizing fields, it solves a problem I don't encounter, so I have no preference.